creditcoin-js/src/examples/ctc/contracts/GluwaGateToken.sol/GluwaGateToken.json

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
This is duplicate with `integration-tests/src/ethereum/ctc/contracts/GluwaGateToken.sol/GluwaGateToken.json`. One of them needs to go.

Reply:
Removed the one in the integration test directory and kept the creditcoin-js one.